eras/byron/ledger/executable-spec/byron-spec-ledger.cabal

```diff
@@ -36,7 +36,7 @@ library
         -Wincomplete-uni-patterns -Wredundant-constraints -Wunused-packages
 
     build-depends:
-        base >=4.14 && <4.19,
+        base >=4.14 && <5,
```

Comment:
Change like this requires a patch version update. Could you please bump up the version of `byron-spec-ledger` in the cabal file and in the CHANGELOG from `1.0.0.1` to `1.0.0.2`